README.md

```diff
@@ -8,19 +8,22 @@ Terraform enables you to safely and predictably create, change, and improve prod
 
 ## Contents
 
-* [Official Resources](#official-resources)
-* [Community](#community)
-* [Books](#books)
-* [Tutorials and Blog Posts](#tutorials-and-blog-posts)
-* [Community Modules](#community-modules)
-* [Private Module Registries](#private-module-registries)
-* [Providers](#providers)
-* [Testing](#testing)
-* [Tools](#tools)
-* [Libraries](#libraries)
-* [Terraform Enterprise](#terraform-enterprise)
-* [Talks](#talks)
-* [Editor Plugins](#editor-plugins)
+- [Awesome Terraform ![Awesome](https://github.com/sindresorhus/awesome)](#awesome-terraform-awesomehttpsgithubcomsindresorhusawesome)
```

Comment:
I guess the only change needed is the addition of the license? The awesome terraform logo is already there.

Reply:
This must be brought in by my editor, sorry about that. I will fix this.